source/fem-dem/ib_particles_dem.cc

```diff
@@ -520,31 +573,71 @@ IBParticlesDEM<dim>::calculate_pw_lubrication_force(
 {
   using numbers::PI;
 
+  struct DefaultDBL_MAX
+  {
+    double value = DBL_MAX;
+  };
+  struct DefaultUINT_MAX
+  {
+    int value = UINT_MAX;
+  };
+  enum lowest_floating_wall_indices
+  {
+    lowest_floating_wall_indices = 1000000
+  };
+
```

Comment:
Yeah clearly if you find yourself defining the struct twice, you need to do it cleanly in the .h...